luma/emulator/device.py

```diff
@@ -6,17 +6,25 @@
 import sys
 import atexit
 import logging
+import string
+import curses
+import collections
+try:
```

Comment:
Not exactly sure, but in py2, I think:
* StringIO.StringIO is byte oriented and works with sys.stdout/stderr
* io.StringIO is unicode oriented and doesn't work with sys.stdout/stderr